src/index.html

```diff
@@ -1,14 +1,35 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="box" id="qa-block">
+      <section class="section">
+        <div class="text">
+          In programming languages, a closure is a technique for implementing lexically scoped name binding in a language with first-class functions. Operationally, a closure is a record storing a function together with an environment.
+        </div>
+      </section>
```

Comment:
watch yesterday's lecture till minute 8 and configure VScode auto-formatting for wrapping lines with more than 80 symbols
https://www.youtube.com/watch?v=ScTyxXA8SA8&list=PL7FuXFaDeEX0sQ0Zhwk2ixfwS1P6e0iuW&index=5